aggie-coding-club/Rev-Registration
# Added fullscreen schedule view

## Description

This adds the ability to make the schedule fullscreen. Screen-shottable/save-as-image schedules coming later. I'm considered of the performance for this tbh - it should be really snappy but it is not and I'm not exactly sure why

## How to test

Create different schedules and fullscreen them, and ensure they look right. Always ensure there is at least 8 hours on the schedule at a time (per the `getFirstAndLastHour` function).

CSCE 482 is a good test case, as it has a 15 minute section in them.

## Screenshots

![fullscreen-schedule-demo](https://user-images.githubusercontent.com/7295783/120120575-eba6fa80-c152-11eb-9b72-1e152322402e.gif)

## Related tasks

I converted #566 to be just screenshot a schedule, so no issue for this one
